starter_kits/C/src/MyBot.c

```diff
@@ -0,0 +1,324 @@
+// First off, here are some defines and includes. I forget what exactly needs what
+#include <stdio.h>
+#include <stdlib.h>
+
+// We include time.h so we can seed the RNG for default "smart" movement with time()
+#include <time.h>
+
+
+
+// Now for our data structures. These are intentionally left rather basic, so add whatever you want to them
+struct ship{
+	unsigned char x, y;
+	unsigned short halite;
+} ships[4096];
+
+struct player{
+	unsigned char dropoffs[64], dropoff_count, ship_count, ship_yard[2];
+	unsigned short ships[1024];
```

Comment:
Will this crash when `ships > 1024`? Can you have > 1024 ships?

Reply:
If an individual player has more than 1024 ships at one time, yes. I don't often see anybody go beyond 100 ships out at a time, the 1024 limit should never, ever be even close to being reached.